src/SIL.LCModel/DomainImpl/OverridesLing_Wfi.cs

```diff
@@ -612,7 +612,7 @@ protected override void OnBeforeObjectDeleted()
 		{
 			var repo = Services.GetInstance<IWfiWordformRepository>() as IWfiWordformRepositoryInternal;
 			foreach (int ws in Form.AvailableWritingSystemIds)
-				repo.RemoveForm(Form.get_String(ws), ws);
+				repo?.RemoveForm(Form.get_String(ws), ws);
```

Comment:
I say we just cast `repo` above rather than not calling remove if it's the wrong type

Reply:
Done